frontend/pages/ManageControlsPage/Scripts/components/ScriptListItem/ScriptListItem.tests.tsx

```diff
@@ -19,14 +20,6 @@ const WINDOWS_SCRIPT: IScript = {
   updated_at: "2021-01-01",
 };
 
-beforeAll(() => {
-  jest.useFakeTimers();
-  jest.setSystemTime(new Date("2025-05-02T00:00:00Z")); // "over 4 years ago" after created_at
-});
-afterAll(() => {
-  jest.useRealTimers();
-});
-
```

Comment:
These were causing this test to hang indefinitely. [Using a more flexible matcher](https://github.com/fleetdm/fleet/pull/33679/files#diff-fa9af152aba227376a3e66578812a42b5403b4331ee464cdd07e46edcab4e29eR82) solved the issue this was trying to solve.